tikaExtractor.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, stdenv, buildMavenRepositoryFromLockFile
+, makeWrapper, maven, jdk11_headless
+, nix-gitignore
+}:
+
+
+let
+  mavenRepository = buildMavenRepositoryFromLockFile { file = ./mvn2nix-lock.json; };
+in stdenv.mkDerivation rec {
+  pname = "tika-extractor";
+  version = "1.1";
+  name = "${pname}-${version}";
+  src = fetchGit{
+    url =https://github.com/ipfs-search/tika-extractor;
+    ref ="main";
+    rev= "e629c4a6362916001deb430584ddc3fdc8a4bf6a";
+      };
+
+  nativeBuildInputs = [ jdk11_headless maven makeWrapper ];
+  buildPhase = ''
+    echo "Building with maven repository ${mavenRepository}"
+    mvn package --offline -Dmaven.repo.local=${mavenRepository} -Dquarkus.package.type=uber-jar
+  '';
+
+  installPhase = ''
+    # create the bin directory
+    mkdir -p $out/bin
+
+    # create a symbolic link for the lib directory
+    ln -s ${mavenRepository} $out/lib
+
+    # copy out the JAR
+    ls -l
```

Comment:
remaining debug statements

Reply:
They are requred to keep the jar in a standard loc as per. Iam using the template from mvn2nix

Reply:
ls -l is just a way to list the files, It doesn't make copies.